Content/Dusts/Glow.cs

```diff
@@ -80,4 +83,104 @@ public override bool Update(Dust dust)
 			return false;
 		}
 	}
+
+	public class PixelatedGlow : ModDust
```

Comment:
Do these behaviors differ from the other Glow dusts? Could this just be parented with an override in predraw?

Reply:
their behaviors are different yea